dev-infrastructure/docs/development-setup.md

```diff
@@ -317,234 +317,130 @@ Then register it with the Maestro Server
-
-1) `podman`, `ocm` cli (latest), and [`yq`](https://github.com/mikefarah/yq) cli (version 4+)
-2) An up-to-date [Clusters Service repo](https://gitlab.cee.redhat.com/service/uhc-clusters-service) cloned down (can also use a fork if you have one)
-
-> If you don't have or want to install `yq`, any steps below using `yq` can be done manually
-
-### Configure and run CS
-
-Option 1: Configure and initialize Cluster Service using the script:
-Run ./dev-infrastructure/local_CS.sh from the root of ARO-HCP repo where "uhc-clusters-service" and "ARO-HCP" repos should be at the same level:
-
-* uhc-clusters-service/
-* ARO-HCP/
-* etc
-
-Option 2: You can follow the below manual steps from the root of the CS repo on our system:
-
-1) Follow [Azure Credentials and Pull Secret for HCP creation](#azure-credentials-and-pull-secret-for-hcp-creation) to fetch `azure-creds.json`.
-
-2) Setup required config files
-
-```bash
-# Setup the development.yml
-cp ./configs/development.yml .
-
-# Setup the azure-runtime-config.json
-# Currently following properties are expected in the file:
-# - `cloudEnvironment` : The Azure cloud environment where Cluster Service is running on.
-#   Possible values are 'AzurePublicCloud', 'AzureChinaCloud' and 'AzureUSGovernmentCloud'.
-cp ./configs/azure/example-config.json ./azure-runtime-config.json
-
-# Get azure-first-party-application-client-id
-# This property needs to be set in the forked development.yml file using the value obtained below
-az ad app list --display-name aro-dev-first-party --query '[*]'.appId -o tsv
-
-
-# Update any required empty strings to 'none'
-yq -i '(.aws-access-key-id, .aws-secret-access-key, .route53-access-key-id, .route53-secret-access-key, .oidc-access-key-id, .oidc-secret-access-key, .network-verifier-access-key-id, .network-verifier-secret-access-key, .client-id, .client-secret) = "none"' development.yml
-
-# Generate a provision_shards.config for port-forwarded maestro ...
-make -C $the_aro_hcp_dir/cluster-service provision-shard > provision_shards.config
-
-# the resulting configuration requires two portforwardings into the service cluster
-kubectl port-forward svc/maestro 8001:8000 -n maestro
-kubectl port-forward svc/maestro-grpc 8090 -n maestro
-
-# Alternatively, update provision shards config with new shard manually
-cat <<EOF > ./provision_shards.config
-provision_shards:
-- id: 1
-  maestro_config: |
+## Creating an ARO HCP Cluster via Cluster Service
+
+### Creating a cluster
+1) Login to your CS deployment
+  - Access your CS deployment locally
+    ```bash
+    kubectl port-forward svc/clusters-service 8000:8000 -n cluster-service
+    ```
+  - Login to your CS deployment
+    ```bash
+    ocm login --url=http://localhost:8000 --use-auth-code
+    ```
+
+2) Create pre-requisite resources for cluster creation
+
+    Replace `resource-group`, `vnet-name`, `nsg-name` and `subnet-name` with any valid names.
+  
+  - Create a resource group for your ARO HCP cluster
```

Comment:
```suggestion
  - Create a resource group for the virtual network that will host your ARO HCP cluster(Data plane nodes).
```
Worker nodes are created under `managed_resource_group_name`, they are not created in this resource group. This resource group only holds Vnet and NSG.

Reply:
that's correct, but this is talking about the usage of the virtual network not the resource group itself. I'll reword it a bit to make this clearer.

Reply:
Thanks @JameelB LGTM 👍